.github/workflows/prerelease_trigger.yml

```diff
@@ -27,3 +27,11 @@ jobs:
       environment: "staging"
       git-tag: "${{ github.event.release.tag_name }}"
     secrets: inherit
+
+  run-QA-suite:
```

Comment:
```suggestion
  run-qa-suite:
```

Reply:
@k-macmillan Happy to make this change, noting that this is how the action is named in the other workflows using it. Do you want me to update those names in this PR, too?

Reply:
renaming all jobs to run-qa-suite